pyscal/scalrecommendation.py

```diff
@@ -208,11 +202,6 @@ def interpolate(
         tagstring = "\n".join(tags)
         if do_gaswater:
             interpolant = GasWater(h=h, tag=tagstring)
-            if gasparameter != parameter:
-                logger.warning(
-                    "Different interpolation parameters for Water and for "
-                    "gas in GasWater, this is maybe not what you want"
-                )
```

Comment:
There is a warning a few lines above that says the same, so this one is dropped.